pkg/dispatcher/hash_dispatcher.go

```diff
@@ -199,7 +196,7 @@ func (d *HashDispatcher) updateConsumerSet() error {
 
 	_ = d.consumerSet.Append(toAddConsumers...)
 	d.consumerSet.RemoveAll(toRemoveConsumers...)
-	log.V(4).Infof("Consumers set for current instance: %s", d.consumerSet.String())
+	log.Infof("Consumers set for current instance: %s", d.consumerSet.String())
```

Comment:
Please lower this log level, I missed this in last change :)

Reply:
done